jobs/config.json

```diff
@@ -9539,6 +9539,15 @@
       "UNKNOWN"
     ]
   },
+  "pull-test-infra-gubernator": {
```

Comment:
not in this PR since we checkout test-infra from head. I'll trigger locally to verify.